Project.toml

```diff
@@ -10,14 +10,16 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
+Aqua = "0.5"
```

Comment:
Maybe the `[compat]` is used for the packages in `[deps]` , and `Aqua = "0.5"` should be located in `test/Project.toml`?

>To avoid breaking test when a new Aqua.jl version is released, it is recommended to add version bound for Aqua.jl in `test/Project.toml`:

https://github.com/JuliaTesting/Aqua.jl#specifying-aqua-version

Reply:
It doesn't really matter, listing it in the main `Project.toml` works to enforce the compatibility bound as well. A `test/Project.toml` often complicates dependency resolution, so there's no point adding one just for compat.

We may check that the compat listed in `Project.toml` in effective:
```julia
julia> using TestEnv; TestEnv.activate();

(jl_S3r3NS) pkg> st Aqua
Status `/tmp/jl_S3r3NS/Project.toml`
  [4c88cf16] Aqua v0.5.5

(jl_S3r3NS) pkg> add Aqua@0.4
   Resolving package versions...
ERROR: empty intersection between Aqua@0.4 and project compatibility 0.5
```

Reply:
I see, thanks. My concern was `Aqua.jl` might be listed in [`B/BandedMatrices/Compat.toml`](https://github.com/JuliaRegistries/General/blob/master/B/BandedMatrices/Compat.toml) because I don't know how smart JuliaRegistrator is..

Reply:
It only lists the compat for the deps. For example, `OffsetArrays` [uses Aqua as well](https://github.com/JuliaArrays/OffsetArrays.jl/blob/master/Project.toml), but https://github.com/JuliaRegistries/General/blob/master/O/OffsetArrays/Compat.toml doesn't list the bound on Aqua.

Reply:
Ah, I see. Thanks for the information!